interfaces/Matlab_Toolbox_Revamp/Class/Mixture.m

```diff
@@ -0,0 +1,328 @@
+classdef Mixture < handle
+    
+    properties
+        mixindex
+        phases
+        T
+        P
+    end
+    
+    methods
+        %% Mixture class constructor
+
+        function m = Mixture(phases)
+            % To construct a mixture, supply a cell array of phases and mole
+            % numbers:
+            %    >> gas = Solution('gas.cti');
+            %    >> graphite = Solution('graphite.cti');
+            %    >> mix = Mixture({gas, 1.0; graphite, 0.1});
+            %
+            % Phases may also be added later using the addPhase method:
+            %    >> water = Solution('water.cti');
+            %    >> addPhase(mix, water, 3.0);
```

Comment:
This is no longer a global function, right? One has to call `mix.addPhase(water, 3.0)`?